Asterix/tests/test_propagations.py

```diff
@@ -8,16 +8,26 @@ def test_mft_centering():
     """
     pdim = 8
     rad = pdim / 2
-    pup = roundpupil(pdim, rad)
+    pup = roundpupil(pdim, rad, center_pos='b')
 
     samp = 4
-    efield = mft(pup, real_dim_input=pdim, dim_output=pdim, nbres=samp)
+    efield = mft(pup, real_dim_input=pdim, dim_output=pdim, nbres=samp, dtype_complex='complex128')
     img = np.abs(efield)**2
 
-    four_equal_pixels = (img[3, 3] == img[3, 4]) and (img[3, 3] == img[4, 3]) and (img[4, 3]
-                                                                                   == img[4, 3]) and (img[4, 4]
-                                                                                                      == img[4, 4])
-    assert four_equal_pixels, "PSF from MFT is not symmetric in four center pixels."
+    # four_equal_pixels = (img[3, 3] == img[3, 4]) and (img[3, 3] == img[4, 3]) and (img[4, 3]
+    #                                                                                == img[4, 3]) and (img[4, 4]
+    #                                                                                                   == img[4, 4])
+    # assert four_equal_pixels, "PSF from MFT is not symmetric in four center pixels."
+
+    np.testing.assert_array_almost_equal(img,
```

Comment:
@johanmazoyer is there a reason you wouldn't use
```python
assert np.allclose(img, np.transpose(img), atol=1e-10), "PSF from MFT is not symmetric (transpose PSF != PSF)"
```
from https://numpy.org/doc/stable/reference/generated/numpy.allclose.html ? It's easier to read with the separate `assert`, and since we're only starting with tests might help for clarity.

Reply:
assert_array_almost_equal is a function uses specifically for assert tests, it return a very complete assertion error when it fails : 
![Screenshot 2022-11-04 at 11 35 30](https://user-images.githubusercontent.com/22528508/199952688-f90b9afb-3e2e-4292-9f23-f178aedd1e37.png)

But I don't mind if you prefer the allclose one

Reply:
I'll leave this decision to you. I just think it might get very confusing to have different test outputs (standard pytest and numpt.testing), especially when we just start rolling out tests. We need to keep things simple.

Reply:
Yes that is a good point ,I replace